app/models/condition.rb

```diff
@@ -13,6 +13,10 @@ def sort_key
     [title_key, timestamp_key, Float::INFINITY].compact.first
   end
 
+  def review_title
+    title.blank? ? "Other" : title
```

Comment:
```suggestion
    title.presence || "Other"
```

> `object.presence` is equivalent to `object.present? ? object : nil`
[\[docs\]](https://api.rubyonrails.org/classes/Object.html#method-i-presence)